src/averageAges.js

```diff
@@ -20,12 +20,42 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+
+  /* create array only with men, if century is passed
+  than it creates a separate array */
+  function getSex(male) {
+    const men = (typeof century === 'undefined')
+      ? (male['sex'] === 'm')
+      : (male['sex'] === 'm')
+    && (Math.ceil(male['died'] / 100) === century);
```

Comment:
```suggestion
    const men = century
      ? (male['sex'] === 'm') && (Math.ceil(male['died'] / 100) === century);
      : (male['sex'] === 'm')
```
так трохи читабельніше